exporter/carbonexporter/go.mod

```diff
@@ -3,8 +3,9 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/carbon
 go 1.13
 
 require (
+	github.com/VividCortex/gohistogram v1.0.0 // indirect
 	github.com/census-instrumentation/opencensus-proto v0.2.1
-	github.com/open-telemetry/opentelemetry-collector v0.2.4-0.20200122010738-1a92a7ea5aea
+	github.com/open-telemetry/opentelemetry-collector v0.2.6
```

Comment:
@pjanotti want to double check if this version of the collector is good for the carbon exporter?

Reply:
Synced offline - this version is good!